### PR TITLE
chore: add timezone to query executed analytic event

### PR DESCRIPTION
**Pull request by rephus**
Closes: https://github.com/lightdash/lightdash/issues/9757

### Description:

image

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by TuringLovesDeathMetal** [comment]
@rephus Can we add the time offset to the timezone field? (if it's not too hard 🤞 ) e.g. `Paris/Madrid (+01:00)`

If it's complicated to add, then just let me know and I'll approve in the current state :)

File: packages/frontend/src/api/csv.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export const downloadCsv = async ({
             columnOrder,
             hiddenFields,
             chartName,
+            timezone: query.timezone ?? undefined,
```
Comment (rephus) [inline packages/frontend/src/api/csv.ts:50]: fix from Bruno

**Comment by rephus** [comment]
> @rephus Can we add the time offset to the timezone field? (if it's not too hard 🤞 ) e.g. `Paris/Madrid (+01:00)`
> 
> If it's complicated to add, then just let me know and I'll approve in the current state :)

Not impossible, but I'll have to generate it myself from the timezone, I don't have the data at the moment. 

Let me try it quickly, I can probably use some dayjs util function.

**Comment by rephus** [comment]
@TuringLovesDeathMetal 
image

**Pull request merged**